### PR TITLE
Meta-apo

### PR DESCRIPTION
**Pull request by qdu-zyf**
@BiocondaBot please update

### CODE REVIEW

File: recipes/meta-apo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source:
+  url: https://github.com/qdu-zyf/meta-apo/archive/v{{ version }}.tar.gz
+  sha256: f18e6ace753f027c698d6e0a547450172af8b00636869bd38fd798d5d82df90d
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (druvus) [inline recipes/meta-apo/meta.yaml:17]: You need to add `make` as a dependency

File: recipes/meta-apo/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+make 
+
+cp -r bin/* ${PREFIX}/bin
+cp -r example ${PREFIX}
```
Comment (druvus) [inline recipes/meta-apo/build.sh:8]: It is preferable to distribute example data outside conda. Maybe include a download function to the software.

File: recipes/meta-apo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  host:
+  run:
+
+test:
+  commands:
```
Comment (druvus) [inline recipes/meta-apo/meta.yaml:22]: you should use commands, such as `meta-apo-train --help`

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot Please merge the branch.

File: recipes/meta-apo/meta.yaml (hunk 1 above)
Comment (qibebt-bioinfo) [reply recipes/meta-apo/meta.yaml:18]: Thank you very much for your advices.

**Review by qibebt-bioinfo — APPROVED** [review]

**Comment by qibebt-bioinfo** [comment]
As advised, we have 
1. added make as a dependency
2. changed the test command
3. moved the test data outside Conda

**Review by qibebt-bioinfo — APPROVED** [review]

**Review by qibebt-bioinfo — APPROVED** [review]

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot update

**Comment by qibebt-bioinfo** [comment]
@druvus  Could you please review our commits again? Thank you so much.

**Review by qibebt-bioinfo — APPROVED** [review]

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**